app/assets/stylesheets/custom.css.scss

```diff
@@ -2,40 +2,9 @@ $blue: #65BEDB;
 $tan: #F7C797;
 $dark-blue: #363691;
 $light-tan: #F7E1CB;
-
-header {
-  min-height: 50px;
-  display: inline;
-}
-
-h1 {
-  font-size: 3em;
-  font-weight: bold;
-  text-align: center;
-  color: $dark-blue;
-}
-
-h2 {
-  font-size: 2em;
-  font-style: italic;
-  font-weight: bold;
-  color: $dark-blue;
-}
-
-.menu {
-  background-color: $blue;
-  @include span-columns(12);
-  min-height: 50px;
-  ul {
-    list-style-type: none;
-    li {
-      display: inline;
-      text-decoration: none;
-      float: right;
-      margin-left: 15px;
-    }
-  }
-}
+$transparent-blue: transparentize($blue, 0.2);
+$white: #ffffff;
+$transparent: transparent;
```

Comment:
I agree with you on this. The reason I assigned it to a variable was because it was picked up by Hound and I feel that a mistake I keep making is not assigning variables which has led to mystery guests. Therefore I felt it was important to assign more variables rather than less. I know there is a balance to find and I am quite on the fence with this one.